contracts/treasury/LiquidationStrategyBase.sol

```diff
@@ -245,9 +248,10 @@ abstract contract LiquidationStrategy is ILiquidationStrategy, PermissionAdmin,
   function isLiquidationEnabledAt(uint256 timestamp) public override view returns (bool) {
     if (timestamp < block.timestamp) return false;
```

Comment:
Redundant

Reply:
logically, it makes sense to return false for past timestamps, since liquidations should be disabled in these cases. 
practically, i dont know if there's a use case / attack vector where past timestamps are used... then it is redundant

Reply:
Removed, only have a function to check for current liquidation enable status (as users can always read the liquidation schedule data to compute)